discord/colour.py

```diff
@@ -396,6 +396,14 @@ def yellow(cls) -> Self:
         .. versionadded:: 2.0
         """
         return cls(0xFEE75C)
+    
+       @classmethod
```

Comment:
```suggestion
   @classmethod
```

Reply:
I think that was wrongly indented or something

Reply:
Ah, I didn't catch that thanks